### ISSUE TITLE
graphing with date

### ISSUE THREAD

**Issue by compsagnathus** [n3-charts/line-chart#462]
I have really been happy with n3 and have been using it for a little over a year.  I am in the process of transitioning to v2 and I am having a really hard time getting date based graphs to work.  What is the format that the date needs to be in?

If I manually set the json data like this I can get it to work

```
$scope.data = {"dataset00":[
      {"x":new Date("06-26-2015"),"val_0":15,"val_1":2,"val_2":3,"val_3":4,"val_4":2,"val_5":4},
      {"x":new Date("06-28-2015"),"val_0":14,"val_1":1,"val_2":3,"val_3":3,"val_4":3,"val_5":4},
      {"x":new Date("07-1-2015"),"val_0":15,"val_1":2,"val_2":2,"val_3":4,"val_4":2,"val_5":5}
]};
```

however, date formats without the new Date() will not work

but my json data is coming from a server and I haven’t had much luck.  Any suggestions?

**Comment by lorem--ipsum** [comment]
Hey, thanks for the interest ! We should reaaaally have a Date example on the webpage. In the meantime, here's an example : http://codepen.io/anon/pen/rejzxb?editors=1010

**Comment by compsagnathus** [comment]
Hey fantastic - that worked great.  Thanks so much!  What I did was once I received the data from the server I iterated through each date entry and created a date object.  I had tried this once before but it wasn't working, but it seems to be working fine now.  (this data set is for survey data)

selectedSurvey.graphDataSettings.data.dataset00.forEach(function(survEntry){
            survEntry.x = new Date(survEntry.x);
        });

thanks again!
